scripts/install-worker.sh

```diff
@@ -203,7 +203,12 @@ EOF
 
 sudo yum install -y device-mapper-persistent-data lvm2
 
-INSTALL_DOCKER="${INSTALL_DOCKER:-true}"
+if [[ ! -v "INSTALL_DOCKER" ]]; then
```

Comment:
style preference: I prefer "if conditional then else", rather than "if not-conditional then else".
so we could remove the ! and swap 207 and 209